src/notification/streamlabs.ts

```diff
@@ -0,0 +1,42 @@
+import {Link, Store} from '../store/model';
+import {Print, logger} from '../logger';
+import {config} from '../config';
+import {URLSearchParams} from 'url';
+import fetch from 'node-fetch';
+
+const {streamlabs} = config.notifications;
+let requestParams: URLSearchParams;
+
+if (streamlabs.accessToken) {
+  requestParams = new URLSearchParams();
+  requestParams.append('access_token', streamlabs.accessToken);
+  requestParams.append('type', streamlabs.type);
+  requestParams.append('image_href', streamlabs.imageHref);
+  requestParams.append('sound_href', streamlabs.soundHref);
+  requestParams.append('duration', streamlabs.duration.toString());
```

Comment:
```suggestion
  requestParams.append('access_token', streamlabs.accessToken);
  requestParams.append('duration', streamlabs.duration.toString());
  requestParams.append('image_href', streamlabs.imageHref);
  requestParams.append('message', streamlabs.text.header);
  requestParams.append('sound_href', streamlabs.soundHref);
  requestParams.append('type', streamlabs.type);
  requestParams.append('user_message', streamlabs.text.subheader);
```

Maybe from previous suggestion?